spec/wibox/test_utils.lua

```diff
@@ -67,68 +83,139 @@ local function draw_layout_comparison(actual, expected)
     end
 
     local function draw_widget(widget)
-        local height = widget._height
-        local width = widget._width
+        -- We need a minimum size to render the data in.
+        -- One line/column on all sides for the boundary.
+        -- Three lines, 12 columns for the text.
+        local min_size = { 14, 5 }
+
+        if widget._width < min_size[1] or widget._height < min_size[2] then
+            too_small = true
+        elseif widget._width > 25 or widget._height > 25 then
```

Comment:
From where do you pull this `{ 25, 25 }` limit?

Reply:
There is very little to base things on.

I guess one could argue that with an overall terminal max width of 80 columns, the width could be 35.
But they really don't need to be that big, so going lower allows for scenarios where people have the terminal in a vertical layout on a small screen. So `min_width + 10` (i.e. up to 5 columns of padding), rounded to a nice number, looks good enough to me.

For the height, I don't see any useful reference, so I just took the same number.

Reply:
btw, i recommend using named constants instead of hardcoded values, which would make the code much easier to understand when reading